### PR TITLE
Simplify _jacoco_offline_instrument.

### PR DESCRIPTION
**Pull request by benjaminp**
resolve_command shouldn't be required here.

### CODE REVIEW

**Comment by johnynek** [comment]
Thanks for the PR.

I have to admit, I don't understand when this should or should not be required. Can you add any links to documentation as to why you think it *shouldn't* be required?

Tests passing are a strong hint, but I'd like to find some documentation that could also guide us.

**Comment by benjaminp** [comment]
I think `resolve_command` is only useful if you pass the first string argument. Then, it can parse out of the `$(location //some/tool)` directives for you and return the requisite tools. But if you already know the executable you want, `resolve_command` is killing a fly with a cannon.

File: scala/private/rule_impls.bzl [hunk 1]
```diff
@@ -1351,10 +1347,9 @@ def _jacoco_offline_instrument(ctx, input_jar):
 
     ctx.actions.run(
         mnemonic = "JacocoInstrumenter",
-        inputs = [in_out_pair[0] for in_out_pair in in_out_pairs] + worker_inputs,
+        inputs = [in_out_pair[0] for in_out_pair in in_out_pairs],
         outputs = [in_out_pair[1] for in_out_pair in in_out_pairs],
-        executable = ctx.attr._code_coverage_instrumentation_worker.files_to_run.executable,
-        input_manifests = worker_input_manifests,
+        executable = ctx.attr._code_coverage_instrumentation_worker.files_to_run,
```
Comment (andyscott) [inline scala/private/rule_impls.bzl]: This seems fine to me. Do you (@johnynek?) know if we've re-enabled the coverage tests in this repo? They were disabled, but I recall seeing a PR to fix/reenable them for current Bazel versions.

**Comment by benjaminp** [comment]
Travis is printing:
```
running test test_coverage_on
Test "test_coverage_on" successful (20 sec) 
```

**Comment by johnynek** [comment]
Thanks!

**Pull request merged**